runtime/src/lib.rs

```diff
@@ -31,6 +31,19 @@ pub enum ClientError {
     WentAway = 0xFFFF_FE03,
 }
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Count)]
+pub enum LeaseReadErr {
+    Rc(NonZeroU32),
```

Comment:
Hm, we could probably make this more specific, but right now I'm not finding a convenient enumeraiton to point to.